itest/tarod_harness.go

```diff
@@ -119,11 +121,15 @@ func newTarodHarness(ht *harnessTest, cfg tarodConfig) (*tarodHarness, error) {
 		return nil, err
 	}
 
-	// We'll modify the config slightly here, since we don't need to use
-	// the hashmail system for integration tests.
+	// Conditionally avoid setting up the hashmail system. Many
+	// tests don't require the hashmail system.
+	//
+	// We can skip setting up the hashmail system by modifying the config.
 	//
 	// TODO(roasbeef): make local aperture instance in future
-	finalCfg.HashMailAddr = ""
+	if !enableHashMail {
+		finalCfg.HashMailAddr = ""
```

Comment:
So we'll start to use another container here? Or launch it in a process before everything happens?

Reply:
PR which adds aperture: https://github.com/lightninglabs/taro/pull/248